### PR TITLE
[ID] Add `Help_Centre/Website`

### PR DESCRIPTION
**Pull request by Nivalyx**
Indonesian translation of the `Help_Centre/Website` wiki page (http://osu.ppy.sh/wiki/en/Help_Centre/Website).

### CODE REVIEW

File: wiki/Help_Centre/Website/id.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+## Masalah-masalah umum
+
+### Saya sudah menunggu hingga lebih dari 2 minggu, namun sampai saat ini saya belum mendapatkan kabar apa-apa seputar tiket layanan dukungan (*support ticket*) milik saya!
+
+**Pada umumnya, hal ini disebabkan oleh banyaknya tiket layanan dukungan (*support ticket*) yang sedang kami proses.**
+
+Apabila kamu ingin mengetahui status tiket layanan dukungan (support ticket) milikmu, kamu dipersilahkan untuk kembali menghubungi tim layanan dukungan kami asalkan masih dalam batas yang sewajarnya. Kamu juga dapat menanyakan hal yang sama melalui akun Twitter [@osusupport](https://twitter.com/osusupport) milik kami.
+
+Dalam **kasus-kasus tertentu** yang utamanya melibatkan pelanggaran-pelanggaran berat, tim layanan dukungan kami dapat memberitahukan kamu bahwa tiket layanan dukungan (support ticket) milikmu tidak akan ditindaklanjuti lebih lanjut. Apabila kamu tidak merasa pernah melakukan pelanggaran apapun, kamu tidak perlu khawatir dan hanya butuh untuk lebih bersabar!
```
Comment (wowcake) [inline wiki/Help_Centre/Website/id.md:36]:
```suggestion
Dalam **kasus-kasus tertentu** yang utamanya melibatkan pelanggaran-pelanggaran berat, tim layanan dukungan kami dapat memberitahukan kamu bahwa tiket layanan dukungan (support ticket) milikmu tidak akan ditindaklanjuti lagi. Apabila kamu merasa tidak pernah melakukan pelanggaran apapun, kamu tidak perlu khawatir dan hanya perlu untuk tetap bersabar menunggu!
```

Comment (wowcake) [reply wiki/Help_Centre/Website/id.md:36]: `tindaklanjuti` with `lanjuti` after that sounds out of place..

File: wiki/Help_Centre/Website/id.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+### Apakah saya dapat memblokir pengguna lain?
+
+**Tentu saja bisa.**
+
+Apabila kamu merasa tersinggung atau terlecehkan dengan pesan-pesan yang dikirim oleh pengguna-pengguna tertentu baik itu melalui pesan pribadi (*private message*), forum, ataupun kanal obrolan dalam permainan (*in-game chat*), kamu dapat memblokir mereka dengan mengikuti langkah-langkah berikut.
```
Comment (wowcake) [inline wiki/Help_Centre/Website/id.md:42]:
```suggestion
Apabila kamu merasa tersinggung atau terlecehkan dengan pesan-pesan yang dikirim oleh pengguna-pengguna tertentu, baik itu melalui pesan pribadi (*private message*), forum, ataupun kanal obrolan dalam permainan (*in-game chat*), kamu dapat memblokir mereka dengan mengikuti langkah-langkah berikut.
```

Comment (wowcake) [reply wiki/Help_Centre/Website/id.md:42]: a comma would be nice there

File: wiki/Help_Centre/Website/id.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+#### Memblokir pengguna lain untuk dapat menghubungimu melalui *in-game chat*
```
Comment (wowcake) [inline wiki/Help_Centre/Website/id.md:44]:
```suggestion
#### Memblokir pengguna lain dari menghubungimu melalui *in-game chat*
```

File: wiki/Help_Centre/Website/id.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+2. Ketik `ignore` atau `pengabaian` pada kolom pencarian (quick-search bar) yang muncul, lalu tuju pilihan `Daftar pengabaian`.
+4. Tambahkan nama-nama pengguna yang ingin kamu blokir ke dalam daftar tersebut. Apabila nama pengguna yang ingin kamu blokir memiliki spasi, ganti semua spasi yang ada dengan garis bawah (sebagai contoh, untuk dapat memblokir *The Gatekeeper* kamu harus mengetikkan *The\_Gatekeeper*).
+
+Ke depannya, kamu tidak akan lagi melihat pesan-pesan dari para pengguna tersebut baik yang dikirim pada kanal-kanal obrolan publik ataupun pada pesan pribadi (*private message*).
+
+Apabila kamu hanya ingin memblokir jenis-jenis pesan tertentu saja (semisal *highlight*, *private message*, atau *public message*), kamu dapat menambahkan @h, @p, atau @c pada nama-nama tertentu dalam daftar pengabaian milikmu. Sebagai contoh, `The_Gatekeeper@c` akan memblokir pesan-pesan pada kanal-kanal obrolan publik dan `The_Gatekeeper@ph` akan memblokir *highlight* dan *private message* yang masuk dari pengguna yang bersangkutan.
+
+#### Memblokir pengguna-pengguna yang tidak ditambahkan sebagai teman untuk dapat mengirimkan pesan pribadi (*private message*) melalui *in-game chat*
```
Comment (wowcake) [inline wiki/Help_Centre/Website/id.md:56]:
```suggestion
#### Memblokir pengguna-pengguna yang tidak ditambahkan sebagai teman dari mengirim pesan pribadi (*private message*) melalui *in-game chat*
```

File: wiki/Help_Centre/Website/id.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+**To naturally hide most public forum posts of a particular user, add them to your Foes list via the UCP.**
+
+1. Head over to the [foes list setting in the UCP](https://osu.ppy.sh/forum/ucp.php?i=zebra&mode=foes) and simply add their username to the list as pictured.
+2. Click submit.
+3. You are done! Most forum posts from that user will no longer be visible to you - it will be as if they are never there. Do note that other users quoting their posts will still be visible.
+
+-->
+
+#### Memblokir pengguna-pengguna yang tidak ditambahkan sebagai teman untuk dapat mengirimkan pesan pribadi (*private message*) melalui forum
```
Comment (wowcake) [inline wiki/Help_Centre/Website/id.md:78]:
```suggestion
#### Memblokir pengguna-pengguna yang tidak ditambahkan sebagai teman dari mengirim pesan pribadi (*private message*) melalui forum
```

**Review by wowcake — CHANGES_REQUESTED** [review]

File: wiki/Help_Centre/Website/id.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+Apabila kamu sebelumnya pernah membuat kolom “saya!” pada laman profil osu! milikmu, kolom tersebut **tidak akan pernah hilang dan akan tetap dapat kamu sunting secara bebas ke depannya** bahkan ketika kamu sudah tidak lagi memiliki [supporter tag](https://osu.ppy.sh/home/support) yang berlaku.
+
+### Konten-konten yang ada pada kolom “saya!” milik saya tiba-tiba menghilang!
+
+**Pada umumnya, hal ini disebabkan karena tim kami menemukan konten-konten pada kolom “saya!” milikmu yang melanggar [peraturan komunitas](/wiki/Rules) yang berlaku.**
+
+Apabila kamu berkeberatan dengan hal ini, kamu dapat membuka tiket layanan dukungan (*support ticket*) baru dengan mengirimkan email pada alamat [accounts@ppy.sh](mailto:accounts@ppy.sh).
+
+Apabila kamu tidak yakin apakah konten yang kamu miliki (atau yang hendak kamu letakkan) pada kolom “saya!” milikmu melanggar peraturan atau tidak, harap hubungi anggota-anggota [Global Moderation Team](/wiki/People/Global_Moderation_Team) untuk mendapatkan arahan dan konfirmasi lebih lanjut.
```
Comment (DoryanR) [inline wiki/Help_Centre/Website/id.md:104]:
```suggestion
Apabila kamu tidak yakin apakah konten yang kamu miliki (atau yang hendak kamu letakkan) pada kolom “saya!” milikmu melanggar peraturan atau tidak, harap hubungi anggota-anggota [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) untuk mendapatkan arahan dan konfirmasi lebih lanjut.
```

Comment (Nivalyx) [reply wiki/Help_Centre/Website/id.md:104]: o this error appears to also be persisting in the english version of the article fwiw, might worth fixing this later on a different pr ig

...but yeah for now i got this fixed in the localized article 👍

File: wiki/Help_Centre/Website/id.md (hunk 1 above)
Comment (Nivalyx) [reply wiki/Help_Centre/Website/id.md:36]: fixed, the ending to the sentence that you suggested feels way too convoluted tho so i'll be keeping what i have right now for that i guess...

File: wiki/Help_Centre/Website/id.md (hunk 2 above)
Comment (Nivalyx) [reply wiki/Help_Centre/Website/id.md:42]: true

File: wiki/Help_Centre/Website/id.md (hunk 3 above)
Comment (Nivalyx) [reply wiki/Help_Centre/Website/id.md:44]: did sth else here (and the subsequent points)

**Comment by Nivalyx** [comment]
fixed the points that you guys brought up, thanks for the review @DoryanR @wowcake 🐇

File: wiki/Help_Centre/Website/id.md (hunk 6 above)
Comment (DoryanR) [reply wiki/Help_Centre/Website/id.md]: I already did it on #5864

**Review by TPGPL — APPROVED** [review]

**Pull request merged**